certbot/tests/crypto_util_test.py

```diff
@@ -181,15 +181,15 @@ def setUp(self):
         super(VerifyCertSetup, self).setUp()
 
         self.renewable_cert = mock.MagicMock()
-        self.renewable_cert.cert = SS_CERT_PATH
-        self.renewable_cert.chain = SS_CERT_PATH
-        self.renewable_cert.privkey = RSA2048_KEY_PATH
-        self.renewable_cert.fullchain = test_util.vector_path('cert_fullchain_2048.pem')
+        self.renewable_cert.cert_path = SS_CERT_PATH
```

Comment:
It is certainly not a big deal, but we are not testing exactly the same logic than before here and in the next modifications for this test module.

Previously we were mocking a `RenewableCert` object instance by assigning string values to internal fields `cert` and so on. Then these internal fields would be accessed directly in the `verify_` methods.

Now we are inserting a mock that slightly diverging from the `RenewableCert` interface instance, since `*_path` are ready-only properties, taking their values from the internal fields. Here the mock contains `*_path` as pure string fields, not read-only properties.

The effect is exactly the same for the test, so it is certainly not worth an additional complexity, but for the record, here a `PropertyMock` (https://docs.python.org/3/library/unittest.mock.html#unittest.mock.PropertyMock) could be used.

Reply:
> The effect is exactly the same for the test, so it is certainly not worth an additional complexity, but for the record, here a PropertyMock (https://docs.python.org/3/library/unittest.mock.html#unittest.mock.PropertyMock) could be used.

Cool. I agree a PropertyMock produces objects with more authentic behavior, but for these tests I don't think it matters or that the extra complexity of multiple PropertyMocks is worth it.